ui/src/Controlpanel.tsx

```diff
@@ -16,7 +16,14 @@ export const ControlPanel = ({ settings, setSettings, reset }: { settings: Setti
             getIndices(),
             getEmbeddingModels(),
         ])
-        const newIndicies = Object.keys(getIndicesResponse).filter((key) => !key.startsWith("."))
+        const newIndiciesMaybeWFeedback = Object.keys(getIndicesResponse).filter((key) => !key.startsWith("."))
+        var newIndicies;
+        if(newIndiciesMaybeWFeedback.includes(FEEDBACK_INDEX_NAME)) {
+            newIndicies = newIndiciesMaybeWFeedback.filter((name) => name !== FEEDBACK_INDEX_NAME)
+        } else {
+            newIndicies = newIndiciesMaybeWFeedback
+            createFeedbackIndex()
+        }
```

Comment:
one liner that might??? do the same thing
```
let newIndicies = newIndiciesMaybeWFeedback.filter((name) => !(name == FEEDBACK_INDEX_NAME && createFeedbackIndex()))
```

Reply:
I think I'm actually gonna leave this for readability, the nested condition can look confusing and we'll want this logic outside the UI anyway